DYNAMODB_ENCRYPTED.py

```diff
@@ -0,0 +1,440 @@
+      And: DynamoDB table is not encrypted
+     Then: Return NON_COMPLIANT
+
+   Scenario: 6
+    Given: Rule Lambda function is not allowed to assume Config rule
+     Then: Return ERROR
+
+   Scenario: 7
+    Given: Any unknown error occurred while executing Config Rule
+     Then: Return ERROR
+'''
+
+import json
+import datetime
+import boto3
+import botocore
+
+##############
+# Parameters #
+##############
+
+# Define the default resource to report to Config Rules
+DEFAULT_RESOURCE_TYPE = 'AWS::DynamoDB::Table'
+
+# Set to True to get the lambda to assume the Role attached on the Config Service (useful for cross-account).
+ASSUME_ROLE_MODE = False
+
+#############
+# Main Code #
+#############
+
+def evaluate_compliance(event, configuration_item, valid_rule_parameters):
+    """Form the evaluation(s) to be return to Config Rules
+
+    Return either:
+    None -- when no result needs to be displayed
+    a string -- either COMPLIANT, NON_COMPLIANT or NOT_APPLICABLE
+    a dictionary -- the evaluation dictionary, usually built by build_evaluation_from_config_item()
+    a list of dictionary -- a list of evaluation dictionary , usually built by build_evaluation()
+
+    Keyword arguments:
+    event -- the event variable given in the lambda handler
+    configuration_item -- the configurationItem dictionary in the invokingEvent
+    valid_rule_parameters -- the output of the evaluate_parameters() representing validated parameters of the Config Rule
+
+    Advanced Notes:
+    1 -- if a resource is deleted and generate a configuration change with ResourceDeleted status, the Boilerplate code will put a NOT_APPLICABLE on this resource automatically.
+    2 -- if a None or a list of dictionary is returned, the old evaluation(s) which are not returned in the new evaluation list are returned as NOT_APPLICABLE by the Boilerplate code
+    3 -- if None or an empty string, list or dict is returned, the Boilerplate code will put a "shadow" evaluation to feedback that the evaluation took place properly
+    """
+    if configuration_item['resourceId'] in valid_rule_parameters:
+        return build_evaluation_from_config_item(
+                configuration_item,
+                'COMPLIANT',
+                annotation="The DynamoDB table \"" + configuration_item['resourceId'] + "\" is whitelisted.")
+
+    status_table = configuration_item["configuration"]["tableStatus"]
+    if status_table == "DELETING":
+        return build_evaluation_from_config_item(configuration_item, 'NOT_APPLICABLE')
+
+    # If ssedescription status is not present or its not 'ENABLED' or 'ENABLING' in
+    # configuration_item, meaning the table is not encrypted, return NON_COMPLIANT
+    # else return COMPLIANT.
+    try:
+        status_kms = configuration_item["configuration"]["ssedescription"]["status"]
+        if status_kms in ['ENABLED', 'ENABLING']:
+            return build_evaluation_from_config_item(configuration_item, 'COMPLIANT')
+    except Exception as e:
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

It appears that you are generically passing an `Exception` object without performing any other operation on it. This may hide error conditions that can otherwise be quickly detected and dispatched. We recommend that you catch a more specific exception. If the code must broadly catch all exceptions, consider logging the stack trace using the `[logging.exception()](https://docs.python.org/3/library/logging.html#logging.exception)` API. For example, 

```
try:
    x = 1 / 0
except ZeroDivisionError as e:
    logging.exception(&#39;ZeroDivisionError: %s&#39;, e)

```